fastai/data/external.py

```diff
@@ -10,13 +10,15 @@
 
 # Cell
 @lru_cache(maxsize=None)
-def fastai_cfg():
+def fastai_cfg() -> Config: # Config that contains default download paths for `'data', 'model', 'storage' and 'archive'`
     "`Config` object for fastai's `config.ini`"
     return Config(Path(os.getenv('FASTAI_HOME', '~/.fastai')), 'config.ini', create=dict(
         data = 'data', archive = 'archive', storage = 'tmp', model = 'models'))
 
 # Cell
-def fastai_path(folder):
```

Comment:
I don't think this one needs docments - improve the docstring instead

Reply:
Slightly improved the docstring